doc/source/whatsnew/v0.25.0.rst

```diff
@@ -495,7 +495,7 @@ Other Deprecations
   Use the public attributes :attr:`~RangeIndex.start`, :attr:`~RangeIndex.stop` and :attr:`~RangeIndex.step` instead (:issue:`26581`).
 - The :meth:`Series.ftype`, :meth:`Series.ftypes` and :meth:`DataFrame.ftypes` methods are deprecated and will be removed in a future version.
   Instead, use :meth:`Series.dtype` and :meth:`DataFrame.dtypes` (:issue:`26705`).
-
+- The ``notebook`` argument of :meth:`DataFrame.to_html` is deprecated (:issue:`23820`).
```

Comment:
mention the replacement

Reply:
there would be no replacement from an end user perspective. see https://github.com/pandas-dev/pandas/pull/26815#issuecomment-501402253